e2e/runner_test.go

```diff
@@ -26,60 +28,38 @@ func testRunner(t *testing.T) {
 		require.NoError(t, err)
 		acknowledgement.WaitForStreamToBeReady(stream)
 
-		resp, err := client.RunnerClient.Create(context.Background(), &pb.CreateRunnerRequest{
-			ServiceHash: testServiceHash,
-			Env:         []string{"BAR=3", "REQUIRED=4"},
-		})
+		msg, err := runBuilder.Start(engineAddress, testServiceStruct, []string{"BAR=3", "REQUIRED=4"})
```

Comment:
I would start the service and then when started, create the message directly in the test with the result of the start (if needed) this way the start is not related to a message just to its function of starting a runner.

Reply:
You could even use the querier `/runner/hash` and you would have all the information to call the functions
- `build(cont container.Container, srv *service.Service, ipfsEndpoint string)`
- `start(cont container.Container, srv *service.Service, instanceHash hash.Hash, runnerHash hash.Hash, imageHash string, env []string, engineName, port string)`

That we can put public. This way the whole LCD struct in the builder package can be removed

Reply:
done